### ISSUE TITLE
In the facelock function in methods.py, there appears to be a sign error in the update step of X_adv.

### ISSUE THREAD

**Issue by barondun** [taco-group/FaceLock#9]
In the _facelock_ function in _methods.py_, there appears to be a sign error in the update step of _X_adv_.

The current code: _X_adv = X_adv + grad.detach().sign() * actual_step_size_
should be changed to: _X_adv = X_adv - grad.detach().sign() * actual_step_size_

The total loss is designed to be minimized (it combines _-loss_cvl_ for attacking identity, _loss_encoder_ for latent consistency, and _loss_lpips_ for perceptual similarity). A gradient descent step therefore requires a minus sign: _X_adv = X_adv - lr * sign(grad)_. The current plus sign performs gradient ascent, which would maximize the total loss, causing the image to drift away from the original and inadvertently make the faces more similar (contrary to the attack goal).
